tests/phpunit/api/v3/GeometryTest.php

```diff
@@ -271,6 +271,16 @@ public function testMySQLSTContains() {
     $this->callAPISuccess('Geometry', 'delete', ['id' => $upperHouseDistrictMBR['id']]);
     $this->callAPISuccess('GeometryType', 'delete', ['id' => $UHGeometryType['id']]);
     $this->callAPISuccess('GeometryCollection', 'delete', ['id' => $UHCollection['id']]);
+
+    // Check that providing an id for either geometry that doesn't exist fails
+   $this->callAPIFailure('Geometry', 'contains', [
+      'geometry_a' => 99999,
+      'geometry_b' => $upperHouseDistrict['id'],
+    ], 'Geometry 99999 Does not exist in the database');
```

Comment:
@unfinishedteleporter need to have a # in front of the numbers as per https://github.com/australiangreens/au.org.greens.civigeometry/pull/40/files#diff-e476444ffc64748061579ed65e3c31a76e69a9f0dad6cf541b9f021bd7dbbe8eR380

Reply:
Yeah, whoops. I also didn't realise it actually checked that message, thought it was just to display it.